nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -608,7 +622,7 @@ in
           };
       } // optionalAttrs cfg.useBootLoader
       { "/boot" =
-          { device = "/dev/vdb2";
+          { device = lookupDriveDeviceName "boot" cfg.qemu.drives;
```

Comment:
This one isn't right yet. It generates `/dev/vdb`, but it should be `/dev/vdb2`, thus breaking `nixos-rebuild switch` inside the VM.

Reply:
I think you accidentally deleted the `2` in https://github.com/NixOS/nixpkgs/pull/92205/commits/710827b2d6157e5636da4b308962e3088575337c#diff-84b6531959e87631dfc70531fbc5bba0L610.

Reply:
So that commit should change it to this instead:

```
{ device = "${lookupDriveDeviceName "boot" cfg.qemu.drives}2"; # 2 for e.g. `vdb2`, as created in `bootDisk`
```

Reply:
Thanks!  Fixed.